frontend/assets/icons/viewer/align.svg.tsx

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Clip should reset if I click on a view that does not have a clip configuration

![24-01-04 02-34-22](https://github.com/3drepo/3drepo.io/assets/11945337/2e2cc8c7-43fb-41e2-a835-7f1b87662565)